docker/set_service_uri.sh

```diff
@@ -22,6 +22,17 @@ else
     LOCAL_ETHEREUM_RPC_URL=${LOCAL_ETHEREUM_RPC_URL:-http://localhost:8545}
 fi
 
+# Read the deployer private key from file
```

Comment:
Good we reuse this here.

I still think we might want to pass all the key in for any real deploy (rather than auto-generate), but at least we are the same in both scripts now